packages/modals/src/styled/StyledDrawerModalBackdrop.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledBackdrop } from './StyledBackdrop';
+
+const COMPONENT_ID = 'modals.drawer_modal.backdrop';
+
+export const StyledDrawerModalBackdrop = styled(StyledBackdrop).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  background-color: transparent;
```

Comment:
The design decision to apply `overflow: hidden` to the `<body>` on Garden's `<Modal>` is based, in large part, on the colored backdrop taking over the screen.

With transparency, it doesn't feel right that the background content is unable to scroll. The implementation should be adjusted to _not_ apply body `overflow: hidden` for the `<Drawer>`-styled modal.

Reply:
The hope is to be less jarring than a standard modal and not block visual access to the page unnecessarily. If a user wants to start interacting with the page content again, it means they no longer want to perform the action they prompted the Drawer with (and it will automatically close on click).

Reply:
> The hope is to be less jarring than a standard modal 

I'm not sure what that means; could you clarify how the "standard modal" is "jarring"? Is the current Garden [Modal](https://zendeskgarden.github.io/react-components/modals/#basic) jarring? And how would full transparency address that issue?

>  The hope is to...not block visual access to the page unnecessarily

I've outlined a couple of reasons why I think it could be necessary above. It seems like you may think those are invalid reasons? I would like to understand why.

> If a user wants to start interacting with the page content again, it means they no longer want to perform the action they prompted the Drawer with (and it will automatically close on click).

Yeah, I understand this. But I'm not sure how this is relevant to the concerns I've posed [here](https://github.com/zendeskgarden/react-components/pull/884#discussion_r492251198)? Also, it isn't clear what _"it will automatically close on click"_ means. How is it automatic, if the user has to initiate the event?

Reply:
By jarring I meant how the standard modal interrupts by blocking vision to the page content and appears in the center of the page. 

I think those concerns have merit and I've shared them at various points while writing the design spec (early iterations had dark backdrops). Users will see the Drawer close when they click outside it and since it's elevated towards to the user on top of the page, the Drawer closing will be an immediate response to a click so I don't foresee any confusion there. 

This interaction is really similar to the Tooltip modal. If a user opens a Tooltip modal and tries to interact with anything else on the page, the first click closes the Tooltip. If this interaction proves confusing to users in practice, I imagine we'll need to revisit and fix both scenarios.

Reply:
> By jarring I meant how the standard modal interrupts by blocking vision to the page content and appears in the center of the page.

Thanks for clarifying as I would not have interpretted this as jarring. Isn't interruption the [purpose](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal) of a modal dialog though? To interrupt the user by presenting UI they must interact with before moving on? 

> This interaction is really similar to the Tooltip modal

This interaction is also similar to the `Modal`. It is difficult to make the code changes when the engineering reason I must provide is "Polaris does it" or "X component does it". I'll go ahead and leave this fully transparent and remove the overflow hidden, but I’m still concerned there aren’t concrete reasons to do so.

Reply:
If the user is able to interact with the background content, that would make the scroll-bar visible. Especially in Windows environments we will start seeing a sidebar sandwich that may not look valid with a full-height modal.

Reply:
From the [W3C design docs](https://www.w3.org/TR/wai-aria-practices/#dialog_modal)

> Windows under a modal dialog are inert. That is, users cannot interact with content outside an active dialog window. Inert content outside an active dialog is typically visually obscured or dimmed 

If their inert scrolling wouldn't be allowed, similar to `Modal` and `TooltipModal`

Reply:
Resolving thread, for now with the full transparency background and scroll lock, to get component released and into usage ✅ . I'll note some of JZ and my concerns and reopen this thread when the topic arises again.